### PR TITLE
HelmInitTask lifecycle messages

### PR DESCRIPTION
**Pull request by wind57**

### CODE REVIEW

File: src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java [hunk 1]
```diff
@@ -34,10 +34,11 @@ public File getTaskOutput() {
 
 	private boolean notNewerThan30(Task task) {
 		String version = getVersion();
-		logger.debug("found version : " + version + " of helm");
 		boolean is30OrNewer = HelmPlugin.isVersion3OrNewer(version);
 		if(is30OrNewer) {
```
Comment (wind57) [inline src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java:38]: let me just say that I would perfectly understand if you would not accept such a PR (and more the like which I plan). The thing is we will run this in a CI/CD environment, with many helm versions from many (I mean hundreds) different projects. Lifecycle messages are a _must_ for me, if I want to be able to somehow get an understanding of what went wrong and where.

File: src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java [hunk 2]
```diff
@@ -34,10 +34,11 @@ public File getTaskOutput() {
...
-			logger.debug("version : " + version + " is higher than 3.0. This operation is a NO-OP");
+			logger.lifecycle("version : " + version + " is higher than 3.0. 'HelmInitTask' is a NO-OP");
```
Comment (jekkel) [inline src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java:39]:
```suggestion
			logger.lifecycle("version : {} is higher than 3.0. 'HelmInitTask' is a NO-OP", version);
```

File: src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java [hunk 3]
```diff
@@ -34,10 +34,11 @@ public File getTaskOutput() {
...
+		} else {
+			logger.lifecycle("will issue : './helm init --client-only' in : " + getHelmHomeDirectory());
```
Comment (jekkel) [inline src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java:41]:
```suggestion
			logger.lifecycle("will issue : './helm init --client-only' in : {}", getHelmHomeDirectory());
```

File: src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java (hunk 2 above)
Comment (wind57) [reply src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java]: @jekkel agreed. thank you

File: src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java (hunk 3 above)
Comment (wind57) [reply src/main/java/com/kiwigrid/k8s/helm/tasks/HelmInitTask.java]: agreed, thank you

**Pull request closed without merging**